### ISSUE TITLE
The rule "MD050 - Strong style" should show problems in a file, but does not.

### ISSUE THREAD

**Issue by sergey-rubtsov-05** [DavidAnson/markdownlint#1132]
Extension `markdownlint` for `Visual Studio Code` does not show "MD050 - Strong style" violations in *.md files.

# Environment

* Visual Studio Code version: `1.86.1` (user setup)
* `markdownlint` extension version: `0.54.0`

## Content of `.markdownlint.json`
```json
{
    "MD050": {
        "style": "underscore"
    }
}
```

# Content of file which can demonstrate the problem

```md
# Strong text example

**Test.** <-- no problem here, but should be

**Test2** other text. <-- no problem here, but should be

**Test3** - other text. <-- no problem here, but should be

* **Test4** - other text. <-- problems here, as expected

```

## Screenshot

![image](https://github.com/DavidAnson/vscode-markdownlint/assets/5280736/ce06ffe2-17dd-4a15-a55c-efea2fda3fa3)

**Comment by sergey-rubtsov-05** [comment]
It was really fast. Thank you. When I can expect a release of vs code extension with the fix?
